src/ai/backend/manager/api/vfolder.py

```diff
@@ -1805,93 +1805,55 @@ async def delete(request: web.Request) -> web.Response:
     return web.Response(status=204)
 
 
-async def leave_request(request: web.Request, params: Any) -> web.Response:
-    """
-    Common request for leaving a shared vfolder.
-    """
-    root_ctx: RootContext = request.app["_root.context"]
-
-    if params["vfolder_id"] is None:
-        raise InvalidAPIParameters("Missing vfolder_id paramater.")
-    if params["user_uuid"] is None:
-        params["user_uuid"] = request["user"]["uuid"]
-
-    vfolder_id = params["vfolder_id"]
-    user_uuid = params["user_uuid"]
-    async with root_ctx.db.begin() as conn:
-        query = (
-            sa.delete(vfolder_permissions)
-            .where(vfolder_permissions.c.vfolder == vfolder_id)
-            .where(vfolder_permissions.c.user == user_uuid)
-        )
-        await conn.execute(query)
-    resp = {"msg": "left the shared vfolder"}
-    return web.json_response(resp, status=200)
-
-
 @auth_required
 @server_status_required(ALL_ALLOWED)
 @vfolder_permission_required(VFolderPermission.READ_ONLY)
-async def leave(request: web.Request, row: VFolderRow) -> web.Response:
-    """
-    Leave a shared vfolder.
-
-    Cannot leave a group vfolder or a vfolder that the requesting user owns.
-    """
-    if row["ownership_type"] == VFolderOwnershipType.GROUP:
-        raise InvalidAPIParameters("Cannot leave a group vfolder.")
-    if row["is_owner"]:
-        raise InvalidAPIParameters("Cannot leave a vfolder owned by the requesting user.")
-
-    access_key = request["keypair"]["access_key"]
-    perm = row["permission"]
-    params = {
-        "vfolder_id": row["id"],
-        "user_uuid": request["user"]["uuid"],
-    }
-    log.info(
-        "VFOLDER.LEAVE(ak:{}, vfid:{}, uid:{}, perm:{})",
-        access_key,
-        params["vfolder_id"],
-        params["user_uuid"],
-        perm,
-    )
-    return await leave_request(request, params)
-
-
-@superadmin_required
-@server_status_required(ALL_ALLOWED)
-@vfolder_permission_required(VFolderPermission.READ_ONLY)
 @check_api_params(
     t.Dict(
         {
-            tx.AliasedKey(["shared_user_id", "sharedUserId"]): t.String,
+            t.Key("shared_user_uuid", default=None): t.String | t.Null,
```

Comment:
Is it safe to remove the alias?
We've left it for the webui's camel casing.
As a frontend developer, if you feel it's safe, it's fine to remove it though.